ci-operator/step-registry/redhat-chaos/pvc-scenarios/prometheus-pod/redhat-chaos-pvc-scenarios-prometheus-pod-commands.sh

```diff
@@ -54,6 +55,18 @@ export ENABLE_ALERTS=False
 telemetry_password=$(cat "/secret/telemetry/telemetry_password")
 export TELEMETRY_PASSWORD=$telemetry_password
 
+#Check if PVC is created
+PVC_CHECK=$(oc get pvc $PVC_NAME -n $TARGET_NAMESPACE --no-headers --ignore-not-found)
+
+if [ -z "$PVC_CHECK" ]; then
+  echo "PVC '$PVC_NAME' does not exist in namespace '$NAMESPACE'."
+  echo "Creating PV and PVC"
+  cluster_monitoring_config
+  echo "Sleeping for 60 seconds for the PV and PVC to be bound"
```

Comment:
We might want to replace sleep with a check that polls every second to see if pv/pvc exists within a certain timeout - 2 mins to make it more robust. Thoughts?